src/git.ts

```diff
@@ -106,31 +121,55 @@ export function checkGitRemote(remoteOrigin: string): boolean {
     return isGitUrl(remoteOrigin.endsWith(".git") ? remoteOrigin : `${remoteOrigin}.git`);
 }
 
+export function convertUrlToProtocol(originalUri: string, format: GitProtocols) {
+    const parsedUri = GitUrlParse(originalUri);
+    switch (format) {
+        case GitProtocols.HTTPS:
+            return parsedUri.toString("https");
+        case GitProtocols.SSH:
+            return parsedUri.toString("ssh");
+        case GitProtocols.DEFAULT:
+        default:
+            return originalUri;
+    }
+}
+
 export const TMP_DIR_PREFIX = "temp_rookout_";
 
+const getProtocolFromStore = () => {
+    const protocol = store.get("gitProtocol", "0");
+    if (isNaN(protocol)) {
```

Comment:
When is it NaN?

Reply:
Not supposed to be but it would break shit if it is